testdata/local/pods.json

```diff
@@ -0,0 +1,2 @@
+{"_id":"localhost", "secret":"de249183582947721fdfb2ea1796574b","agent_revision":"foo", "type": "agent", "status": "running", "task_creation_opts": {"env_secrets": {"POD_SECRET": "de249183582947721fdfb2ea1796574b"}}}
+{"_id":"localhost2", "secret":"de249183582947721fdfb2ea1796574b","agent_revision":"foo", "type": "agent", "status": "running"}
```

Comment:
I see the `localhost` doc is used for the agent running the smoke test, but what's the second pod document for?

Reply:
It's not used for anything I just felt the need to insert multiple pod documents for some reason. Removing.